src/clj/game/core/runs.clj

```diff
@@ -289,20 +298,18 @@
                     "Card from hand"
                     (if-let [accessed (some #(when-not (contains? already-accessed %) %)
                                             (shuffle (get-in @state [:corp :hand])))]
-                      (do (system-msg state side (str "accesses " (:title accessed)))
-                          (when-completed (handle-access state side [accessed])
-                                          (let [from-root (get-root-content state)]
-                                            (if (or (< 1 from-hq) (not-empty from-root))
-                                              (continue-ability
-                                                state side
-                                                (access-helper-hq state (dec from-hq) (conj already-accessed accessed))
-                                                card nil)
-                                              (effect-completed state side eid)))))
+                      (when-completed (msg-handle-access state side [accessed])
+                                      (let [from-root (get-root-content state)]
+                                        (if (or (< 1 from-hq) (not-empty from-root))
+                                          (continue-ability
+                                            state side
+                                            (access-helper-hq state (dec from-hq) (conj already-accessed accessed))
+                                            card nil)
+                                          (effect-completed state side eid))))
                       (effect-completed state side eid nil))
                     ;; accessing a rezzed upgrade
                     (let [accessed (some #(when (= (:title %) target) %) (get-root-content state))]
-                      (system-msg state side (str "accesses " (:title accessed)))
-                      (when-completed (handle-access state side [])
+                      (when-completed (msg-handle-access state side (make-eid state) [] (:title accessed))
```

Comment:
actually it's not that ... this was an account siphon run with no upgrades.

Seamus spends  to play Account Siphon.
Bruno has no further action.
Seamus continues the run.
Bruno rezzes Resistor.
Seamus spends 4  to install Paperclip.
Bruno has no further action.
Seamus continues the run.
Bruno
-1 to break
Seamus sets credit to 7 (-1).
Bruno spends 6  to rez SanSan City Grid.
Bruno has no further action.
Seamus uses Desperado to gain 1 .
Seamus accesses Project Beale.
Seamus steals Project Beale and gains 2 agenda points.
**Seamus accesses .**
Seamus spends  to gain 1 .